corehq/apps/hqadmin/views.py

```diff
@@ -682,29 +682,35 @@ def loadtest(request):
     template = "hqadmin/loadtest.html"
     return render(request, template, context)
 
-@require_superuser
-def doc_in_es(request):
-    doc_id = request.GET.get("id")
-    if not doc_id:
-        return render(request, "hqadmin/doc_in_es.html", {})
 
-    couch_doc = {}
+def _lookup_id_in_couch(doc_id):
     db_urls = [settings.COUCH_DATABASE] + settings.EXTRA_COUCHDB_DATABASES.values()
     for url in db_urls:
         try:
-            couch_doc = Database(url).get(doc_id)
-            break
+            doc = Database(url).get(doc_id)
         except ResourceNotFound:
             pass
-    query = {"filter":
-                {"ids": {
-                    "values": [doc_id]}}}
+        else:
+            return {
+                "doc": json.dumps(doc, indent=4, sort_keys=True),
+                "doc_id": doc_id,
+                "doc_type": doc.get('doc_type', 'Unknown'),
+                "db_url": url.split('@')[-1],
```

Comment:
awesome, I'll do that - I wasn't crazy about displaying even part of this setting.